### DISCUSSION TITLE
Why can't it be used like this now?

### DISCUSSION THREAD

**Discussion by blinlincom** [Sketchware-Pro/Sketchware-Pro#678] [Q&A]
![mmexport1671267917804](https://user-images.githubusercontent.com/38490701/208236015-93f0efac-eebc-444c-9cbc-541bcd0d46cc.jpg)
![Screenshot_2022-12-17-17-01-18-110_com sketchware remod](https://user-images.githubusercontent.com/38490701/208236032-6fc06e28-8e21-439c-9eaa-05af89519076.jpg)

**Comment by j-px** [comment]
Have a look at v6.4.0-rc05's changelog. The custom application class now needs a static `getContext()` method returning the application context. Have a look at the new default SketchApplication.java for an idea how to implement it.

Otherwise you could just disable debug logcat logs in Build Settings.

**Reply by blinlincom** [reply]
Sket chApplication. Java where to find it

**Reply by j-px** [reply]
Create a new project, run it, and then check in /Internal storage/.sketchware/mysc/\<project ID\>/app/src/main/java/…

**Reply by blinlincom** [reply]
Can you write an example? I haven't understood it all the time.

**Reply by j-px** [reply]
Here's an example SketchApplication.java (renamed to .txt, thanks GitHub), found at /Internal storage/.sketchware/mysc/647/app/src/main/java/com/my/newproject/SketchApplication.java.

[SketchApplication.java.txt](https://github.com/Sketchware-Pro/Sketchware-Pro/files/10307861/SketchApplication.java.txt)
